### PR TITLE
Add gifffer@1.4.0 w/ git auto-update

### LINKED ISSUE #9635: [Request] Add gifffer
**Library name:** gifffer
**Git repository url:**https://github.com/krasimir/gifffer
**npm package url(optional):** 
**License(s):**MIT
**Official homepage:**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster than just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by x09326**
Pull request for issue: #9635 
Related issue(s): # #
@kennynaoh Please help me review it, thanks.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/krasimir/gifffer
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars:498
   - Count of watchers:20
   - Count of forks:48
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9635, cc @krasimir

### CODE REVIEW

File: ajax/libs/gifffer/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "url": "git@github.com:krasimir/gifffer.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/jashkenas/underscore.git",
+    "fileMap": [
+      {
+        "basePath": "build",
+        "files": [
+          "gifffer*"
```
Comment (kennynaoh) [inline ajax/libs/gifffer/package.json]: I think it's better to use `**/*` here.

**Comment by x09326** [comment]
@kennynaoh 
I have modified it, please help me review it, thanks.

**Comment by kennynaoh** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
@kennynaoh you should approve the PR first.

File: ajax/libs/gifffer/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    "gif",
+    "player"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:krasimir/gifffer.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/jashkenas/underscore.git",
```
Comment (PeterDaveHello) [inline ajax/libs/gifffer/package.json]: Is this repo correct?

**Comment by kennynaoh** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
@x09326 @kennynaoh what's the reason why there is no any mention about the repo url fix?

**Comment by x09326** [comment]
Sorry I forgot to mention that,

**Pull request merged**